addons/web/static/src/core/py_js/py_interpreter.js

```diff
@@ -270,6 +270,9 @@ const DICT = {
 export function evaluate(ast, context = {}) {
     const dicts = new Set();
     let pyContext;
+    if ('context' in context) {
```

Comment:
this feels wrong. i don't think you should do that (also => test)

Reply:
They are a test => https://github.com/odoo/odoo/pull/104741/files?show-viewed-files=true&file-filters%5B%5D=#diff-458d6b158302a069202688cc5f5e609b08a7d8a0ad26aa1383bd29b2bada945aR99

We must force the context as dict to use "get", also it's used like an object without get